tag.php

```diff
@@ -64,8 +66,24 @@
 		}
 		$context['page_category'] = 'Tag Page';
 
+		if ( ListingPagePagination::is_active() ) {
+			$view = ListingPageGridView::is_active() ? 'grid' : 'list';
+
+			$query_template = file_get_contents( get_template_directory() . "/parts/query-$view.html" );
+
+			$content = do_blocks( $query_template );
+
+			$context['query_loop'] = $content;
+		}
+
 		$campaign = new TaxonomyCampaign( $templates, $context );
 
+		if ( ListingPagePagination::is_active() ) {
```

Comment:
For now I just made it render with the mystic `TaxonomyCampaign` class. We can remove this class once we're sure we can remove the old path.